dtm0/dtx.c

```diff
@@ -336,68 +343,88 @@ static void dtx_exec_all_ast_cb(struct m0_sm_group *grp, struct m0_sm_ast *ast)
 static void dtx_persistent_ast_cb(struct m0_sm_group *grp,
 				  struct m0_sm_ast   *ast)
 {
+	struct m0_dtm0_log_rec       *rec;
+	struct m0_dtm0_dtx           *dtx;
 	struct m0_dtm0_pmsg_ast      *pma = ast->sa_datum;
-	struct m0_dtm0_dtx           *dtx = pma->p_dtx;
```

Comment:
If "->p_dtx" is not used in this place, can we remove it from the "struct m0_dtm0_dtx"?

Reply:
I see it is already replaced by a pointer of log.

Reply:
Yes.

Reply:
OK